src/sqllogictest/src/bin/sqllogictest.rs

```diff
@@ -326,21 +325,22 @@ async fn rewrite(config: &RunConfig<'_>, args: Args) -> ExitCode {
                 Ok(entry) => {
                     if entry.file_type().is_file() {
                         if let Err(err) = runner::rewrite_file(&mut runner, entry.path()).await {
```

Comment:
I don't like that we have to make the adjustment in two different places; it shows that we have a code clone here.

Though, I don't see cleaning that up in the scope of this PR.